### ISSUE TITLE
Remove/extract New Relic middleware

### ISSUE THREAD

**Issue by PetrGlad** [zalando-stups/swagger1st#69]
The New Relic dependency is out of scope of the project and is not required in most cases. OTOH it can be useful as a separate Ring middleware somewhere else. So I suggest to extract New Relic adapter into separate library project.
I can do code changes but since I do not use New Relic I would be unable to verify that it still works.
As an alternative I can prepare the PR that just removes New Relic adapter.

**Comment by sebastianpoeplau** [comment]
I agree that it should be factored out, but I'm not a New Relic user either. Maybe @harti2006 or @prayerslayer can comment...
